eval/eval_tracker.py

```diff
@@ -317,6 +317,18 @@ def insert_eval_results(
             session.rollback()
             raise RuntimeError(f"Database error in insert_eval_results: {str(e)}")
 
+    def check_if_already_done(self, name: str, model_id: uuid.UUID):
+        with self.session_scope() as session:
+            rows = session.query(EvalResult).filter_by(model_id=model_id).all()
+            if not rows:
+                return False
+
+            for row in rows:
+                eval_setting = session.query(EvalSetting).filter_by(id=row.eval_setting_id).first()
+                if name in eval_setting.name:
+                    return True
+            return False
```

Comment:
Does this handle the ai2_arc naming thing?

Reply:
(Flagging here that it doesn't work for tasks like ai2_arc where the name isn't an exact substring of the name in the DB. Should still be good to merge)